### ISSUE TITLE
How to update glider DAC plugin

### ISSUE THREAD

**Issue by kerfoot** [ioos/compliance-checker#989]
**Version of compliance checker running:**

Latest version

**Describe the checker this affects:**
_The ACDD, IOOS, and CF checkers are currently considered in scope for the purposes of the issue tracking on this repository. Issues with other plugins should be placed into their respective repositories._

Glider DAC 3.0 plugin

**Attach a minimal CDL or NetCDF file which is able to reproduce the issue**

**To Reproduce:**
_List the steps or command line to run to reproduce the issue_

**Describe the issue below:**
_Any additional detail or information relevant to the issue_

cf conventions state that all units should be strings:

https://cfconventions.org/Data/cf-conventions/cf-conventions-1.7/build/ch03.html

Practical salinity units should be a string "1", but the glider DAC plugin is not flagging practical salinity units of int 1.  This forces the user to pass xr.open_dataset(nc_file, decode_cf=False) in order to open the data set using xarray. Not sure if this is the case for all CF checkers in the compliance checker.

What is the process for updating the compliance checker and/or glider DAC 3.0 plugin to ensure that the units value is a string?

**Comment by benjwadams** [comment]
You'll want to go to the repo for the Glider DAC Compliance Checker plugin, which is separate from this repo and is located at https://github.com/ioos/cc-plugin-glider .   However, the Glider DAC plugin reuses some code from portions of this repo, especially CF, as I recall.   I'll check that it's not something upstream in CF that could be fixed.

**Comment by benjwadams** [comment]
FYI, run against current master, CF checker does detect the units issue with salinity variable:

```
§3.1 Units
* units (1) attribute of 'salinity' must be a string compatible with UDUNITS
```

**Comment by benjwadams** [comment]
There are currently no checks against the `units` attribute in `cc-plugin-glider`.  I have confirmed that the issue does not lie with the upstream checkers contained within this repository -- a likely fix would be to add the method called by the CF checkers used here. Please create a new issue in the `cc-plugin-gliders` repository linked above.
